Source/ACE/Entity/WorldObject.cs

```diff
@@ -72,6 +85,10 @@ public ushort ForcePositionIndex
 
         public SequenceManager Sequences { get; }
 
+        public object InventoryMutex => this.inventoryMutex;
```

Comment:
I'm really hesitant to think a mutex needs to be public.  Red flag for other code being nosey into this class.